packages/server/src/api/routes/application.js

```diff
@@ -12,6 +12,7 @@ router
     authorized(BUILDER),
     controller.fetchAppPackage
   )
+  .put("/api/:applicationId/appPackage", authorized(BUILDER), controller.update)
```

Comment:
just `api/:applicationId` is probably fine here - the `appPackage` refers to a larger config item